tests/formulas/exoneration_cotisations_employeur_tode.yaml

```diff
@@ -0,0 +1,118 @@
+        chomage_employeur: -1 * (1603.15 * 1.2) * 0.0405  # sous 4 PSS
+  output:
+    exoneration_cotisations_employeur_tode_eligibilite: true
+    # -625.61 = -1 * 1603.15 * 1.2 * (0.07 + 0.0345 + 0.0116 + 0.001 + 0.0855 + 0.019 + 0.0472 + 0.0129 + 0.003 + 0.0405)
+    exoneration_cotisations_employeur_tode: -625.61
+
+
+- name: Exonération partielle de TO-DE pour salarié agricole à rémunération mensuelle brute < 1.6 SMIC
+  period: 2022-04
+  relative_error_margin: 0.002  # fnal : 0.001 -> 0.002
+  input:
+    secteur_activite_employeur: agricole
+    effectif_entreprise: 10  # moins de 50 salariés (seuil fnal 2020+)
+
+    contrat_de_travail_type: cdd
+    contrat_duree_determinee_type: contrat_saisonnier
+    categorie_salarie: prive_non_cadre
+    taches_salarie_type: prolongement_production
+    exposition_accident: faible
+
+    salaire_de_base: 1603.15 * 1.5  # 2404.72
+    hsup: 0
+
+    allegement_cotisation_maladie_mode_recouvrement: anticipe  # maladie
+    cotisation_sociale_mode_recouvrement: mensuel_strict  # famille
+
+  output:
+    assiette_cotisations_sociales: 1603.15 * 1.5
+
+    mmid_employeur: -1 * 1603.15 * 1.5 * 0.13  # cotisation avant allègement
+    assiette_allegement: 1603.15 * 1.5
+    allegement_cotisation_maladie: (1603.15 * 1.5) * (0.13 - 0.07)  # sous 2.5 SMIC
+
+    famille: -1 * 1603.15 * 1.5 * 0.0525  # FIXME: à <= 3.5 SMIC devrait être au prelevements_sociaux.cotisations_securite_sociale_regime_general.famille.taux_reduit
+    accident_du_travail: -1 * 1603.15 * 1.5 * 0.009  # FIXME: taux 2018 à mettre à jour
+    fnal: -1 * (1603.15 * 1.5) * 0.001  # sous 1 PSS (3428€)
+    vieillesse_plafonnee_employeur: -1 * (1603.15 * 1.5) * 0.0855  # sous 1 PSS
+    vieillesse_deplafonnee_employeur: -1 * (1603.15 * 1.5) * 0.019  # sur totalité de la rémunération
+    agirc_arrco_employeur: -1 * (1603.15 * 1.5) * 0.0472  # sous 1 PSS
+    contribution_equilibre_general_employeur: -1 * (1603.15 * 1.5) * 0.0129  # sous 1 PSS
+    contribution_solidarite_autonomie: -1 * (1603.15 * 1.5) * 0.003
+    chomage_employeur: -1 * (1603.15 * 1.5) * 0.0405  # sous 4 PSS
+
+    exoneration_cotisations_employeur_tode_eligibilite: true
+    # -819.05 = -1 * 1603.15 * 1.5 * (0.07 + 0.0525 + 0.009 + 0.001 + 0.0855 + 0.019 + 0.0472 + 0.0129 + 0.003 + 0.0405)
+    exoneration_cotisations_employeur_tode: 1.2 * (-819.05 / 0.4) * (1.6 * 1603.15 / (1603.15 * 1.5) - 1)
+
+
+- name: Exonération nulle de TO-DE pour salarié agricole à rémunération mensuelle brute >= 1.6 SMIC
+  period: 2022-04
+  relative_error_margin: 0.001
+  input:
+    secteur_activite_employeur: agricole
+    effectif_entreprise: 10  # moins de 50 salariés (seuil fnal 2020+)
+
+    contrat_de_travail_type: cdd
+    contrat_duree_determinee_type: contrat_vendanges
+    categorie_salarie: prive_cadre
+    taches_salarie_type: prolongement_production
+    exposition_accident: faible
+
+    salaire_de_base: 1603.15 * 2.6  # 4168.19
```

Comment:
- Le montant des paramètres est 1603.12 et non 1603.15. Pourquoi cet écart ?
- Pour `* 2.6` et pas `* 1.6` ?

Reply:
> Pourquoi * 2.6 et pas * 1.6 ?

Effectivement, l'exonération de TO-DE est nulle à partir de 1.6 SMIC mais on ne sort pas pour autant de toutes les réductions de cotisations employeur. Ce n'était pas obligatoire pour ce test mais afin d'avoir un cas plus neutre, on choisit ici de sortir de toutes les exonérations, la TODE et les autres. 

Celle s'appliquant au seuil le plus haut ici est a priori la [réduction du taux de cotisations maladie : réduction de 6% jusqu'à 2.5 SMIC](https://www.urssaf.fr/portail/home/employeur/calculer-les-cotisations/les-taux-de-cotisations/la-cotisation-maladie---maternit/reduction-de-6-points-du-taux-de.html). C'est pourquoi les ressources choisies ici sont à 2.6 SMIC.

Reply:
> Le montant des paramètres est 1603.12 et non 1603.15. Pourquoi cet écart ?
Dans sa section `2/ Comment calculer l'exonération ?`, la [fiche MSA sur la TO-DE](https://www.msa.fr/lfp/employeur/exonerations-travailleurs-occasionnels?p_p_id=com_liferay_journal_content_web_portlet_JournalContentPortlet_INSTANCE_TnUJJSlWXJvY&p_p_lifecycle=0&p_p_state=normal&p_p_mode=view&_com_liferay_journal_content_web_portlet_JournalContentPortlet_INSTANCE_TnUJJSlWXJvY_read_more=2) indique : 
```
Précisions sur ce calcul

La rémunération mensuelle brute correspond à celle retenue pour le calcul des cotisations de la réduction générale des cotisations patronales. (Réduction générale à l'article L. 241-13 du code de sécurité sociale dans sa version en vigueur en 2010).
 
_Le montant mensuel du SMIC est égal à 151,67 fois le SMIC horaire._
Il doit être corrigé pour un salarié dont la rémunération contractuelle n'est pas fixée pour l'ensemble du mois considéré sur la base d'une durée hebdomadaire (ou rapportée à la durée du cycle) de 35 heures ou d'une durée annuelle de 1607 heures (ex : temps partiel, salarié non mensualisé, salariés dont la rémunération est calculée sur une base ne correspondant pas à la durée de 35 h hebdomadaire ou 1607h par an, situations d'entrées-sorties en cours de mois, salariés dont le contrat de travail est suspendu en raison de maladie avec maintien ou non de salaires, etc.).

Après correction éventuelle du SMIC, cette valeur ne doit pas être augmentée des heures supplémentaires et complémentaires non majorées.
```
Or, en avril 2022 (période des tests), le SMIC horaire brut est de [10.57€](https://github.com/openfisca/openfisca-france/blob/7ab7bceb116707b41c59be8519648787ff2da16c/openfisca_france/parameters/marche_travail/salaire_minimum/smic/smic_b_horaire.yaml#L234) et si on lui applique le facteur cité ci-dessus, nous obtenons : 
10.57 * 151.67 = 1603.1519, soit le 1603.15 utilisé dans les tests.